src/content/posts/big-gabby.md

```
@@ -0,0 +1,12 @@
+---
+title: Big Gabby
+video:
+videoDimensions: 560x315
+orientation: landscape 
+---
+
+You'll fall in love, I know you will
+Our number 6 is the king of Brazil 
```

markdownlint-cli2 (0.18.1)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)